components/server/src/user/user-deletion-service.ts

```diff
@@ -47,6 +50,22 @@ export class UserDeletionService {
             log.debug({ userId: id }, "Is deleted but markDeleted already set. Continuing.");
         }
 
+        if (this.config.enablePayment) {
```

Comment:
This should be governed by BillingMode. But out-of-scope here, and works as well atm. :+1:

Reply:
[ticketed](https://linear.app/gitpod/issue/WEB-191/control-stripe-subscription-deletion-with-billingmode)